source/module_hamilt_lcao/module_gint/grid_bigcell.cpp

```diff
@@ -28,13 +25,9 @@ Grid_BigCell::~Grid_BigCell()
 	// delete tau positions.
 	if(this->flag_tib)
 	{
-		for(int i=0; i<this->nat; i++)
-		{
-			delete[] tau_in_bigcell[i];
-		}
-		delete[] tau_in_bigcell;
+		tau_in_bigcell.clear();
 	}
-	delete[] index_atom;
+	index_atom.clear();
```

Comment:
In C++, the `clear()` function of `std::vector` is used to remove all elements from the container, which reduces the size of the `vector` to 0. However, calling `clear()` does not change the capacity of the `vector`, which is the amount of allocated space in memory. If you want to release memory, you can use the `shrink_to_fit` function or the swap trick.

Reply:
> If you want to release memory

`resize(0)` would be just fine

Reply:
It has already been resolved.